e2e-tests/playwright/package.json

```diff
@@ -1,33 +1,34 @@
 {
     "scripts": {
-        "test": "PW_SNAPSHOT_ENABLE=true playwright test",
+        "test": "NODE_OPTIONS='--no-experimental-fetch' PW_SNAPSHOT_ENABLE=true playwright test",
         "percy": "PERCY_TOKEN=$PERCY_TOKEN PW_PERCY_ENABLE=true percy exec -- playwright test --project=chrome --project=iphone --project=ipad",
         "tsc": "tsc -b",
         "lint": "eslint . --ext .js,.ts",
         "prettier": "prettier --write .",
         "check": "npm run tsc && npm run lint && npm run prettier",
         "codegen": "playwright codegen $PW_BASE_URL",
-        "test-slomo": "PW_SNAPSHOT_ENABLE=true PW_HEADLESS=false PW_SLOWMO=1000 playwright test",
+        "playwright-ui": "npx playwright test --ui",
```

Comment:
I usually run the UI in headless mode.

Since I can see the actions and steps in the UI, I choose to hide all those raised windows (taking the focus). But I also understand that is more a personal choice than the only way to do it.

Reply:
Thanks for the feedback. Yup, definitely have the same workflow as well. I changed the default to headless and other users may set `PW_HEADLESS=false` if they prefer to run in headed mode.